lutris/runners/scummvm.py

```diff
@@ -177,3 +177,6 @@ def get_game_list(self):
             if game.startswith("-----"):
                 game_list_start = True
         return game_array
+
+    def get_main_file(self):
+        return None
```

Comment:
Why None?

Reply:
When I first looked through I didn't see a proper path to the game file.  Looking again, it might be `self.path`, but that appears to be a location for multiple game files and not to a particular entry.  Without knowing what the main file path was, `None` seemed the safest option.